scripts/gulp/lint.js

```diff
@@ -52,5 +64,6 @@ gulp.task( 'lint:scripts', function() {
 gulp.task( 'lint', [
   'lint:build',
   'lint:tests',
-  'lint:scripts'
+  'lint:scripts',
+  'lint:less'
```

Comment:
TRIVIAL: Prolly would name this `lint:styles` to fit with `build:styles` etc. used elsewhere since we go with a tech agnostic approach for these build tasks.

Reply:
Agreed. Will update.